lib/server.js

```diff
@@ -137,13 +140,17 @@ function handleResponse (opts, req, resp, response) {
 function handleRequest (opts, req, resp) {
     var newReq;
 
+    // set the request ID early on for external requests
+    rbUtil.requestId(req);
```

Comment:
it's server.js , `handleRequest()` is called only form the outside. The request id needs to be set as it's used for logging.